cmake/projects/libxml2/hunter.cmake

```diff
@@ -35,9 +36,15 @@ hunter_add_version(
 set(_libxml_unrelocatable_text_files PACKAGE_UNRELOCATABLE_TEXT_FILES "lib/pkgconfig/libxml-2.0.pc")
 hunter_configuration_types(libxml2 CONFIGURATION_TYPES Release)
 if (MSVC)
+    hunter_check_toolchain_definition(NAME "_DLL" DEFINED _hunter_vs_md)
     hunter_pick_scheme(DEFAULT url_sha1_libxml2_msvc)
     # pkgconfig file not applicable
     set(_libxml_unrelocatable_text_files "")
+    hunter_cmake_args(
+        libxml2
+            CMAKE_ARGS
+                BOOST_BUILD_DYNAMIC_VSRUNTIME=${_hunter_vs_md}
```

Comment:
I guess name should be `LIBXML2_BUILD_DYNAMIC_VSRUNTIME`

Reply:
Fixed. Thanks.